external/io_pnetcdf/wrf_io.F90

```diff
@@ -1257,7 +1257,10 @@ SUBROUTINE ext_pnc_open_for_write_begin(FileName,Comm,IOComm,SysDepInfo,DataHand
 !     if(newFileName(i:i) == '-') newFileName(i:i) = '_'
      if(newFileName(i:i) == ':') newFileName(i:i) = '_'
   enddo
-  stat = NFMPI_CREATE(Comm, newFileName, IOR(NF_CLOBBER, NF_64BIT_OFFSET), info, DH%NCID)
+! Thomas changed to NF_64BIT_DATA for CDF5 standard!!!                                                                              
+  stat = NFMPI_CREATE(Comm, newFileName, IOR(NF_CLOBBER, NF_64BIT_DATA), info, DH%NCID)
```

Comment:
Rather than hard-wiring `NF_64BIT_DATA` here, we could choose how to call `NFMPI_CREATE` based on a flag that could be set in the `SysDepInfo` argument that is passed to the `ext_pnc_open_for_write_begin` routine. The `open_w_dataset` routine appears to have access to namelist records and the sysdepinfo string, which is then passed to `wrf_open_for_write_begin` and then on to `ext_pnc_open_for_write_begin`, so it may not be too much work to have a namelist option control whether we use `NF_64BIT_OFFET` or `NF_64BIT_DATA`.